test/ArborXTest_TreeTypeTraits.hpp

```diff
@@ -17,7 +17,7 @@
 #include <tuple>
 
 // NOTE Because std::tuple does not take template template parameters
-template <template <class> class...>
+template <template <class...> class...>
```

Comment:
Did you consider defining an alias that only exposes one template parameter instead of changing this?

Reply:
I've added a single template parameter alias in `CMakeLists.txt` for `ArborX::BVH`, and that worked.